code/code/disc/disc_psionics.cc

```diff
@@ -148,9 +147,9 @@ int TBeing::doPTell(const char *arg, bool visible){
     return FALSE;
   }
 
-  sstring garbed = garble(vict, message, Garble::SPEECH_TELL);
```

Comment:
Did you just remove _all_ garbles from _everywhere_?

Reply:
Negative.  This is only the garble for **psionic** messages.  We have plenty of examples in fantasy of telepathic communications bypassing language barriers - from my perspective, it follows that they should also bypass accents as well.